### ISSUE TITLE
Editable text area in radio label not read in "focus mode"

### ISSUE THREAD

**Issue by ZachPeters** [nvaccess/nvda#6433]
When NVDA is in focus mode on a radio button it will not read the entire label. It fails to read the editable text area nested inside the label. So if a user tabs through the radio buttons they will not be informed of a text field with the radio button label.

Here is a JSBin with an example: http://jsbin.com/wematigifi/edit?html,css

Steps to reproduce: 
1. Tab to the radio group and enter focus mode.
2. Arrow to the radio button with the text area next to it.
3. Notice that NVDA does not read out "Edit" to notify the user there is an editable text area with the radio button.
4. Note: If you arrow to the radio button with the text area and are not in focus mode, NVDA will read out "Edit" to notify the user there is an editable text area with the radio button.

**Comment by jcsteh** [comment]
When in focus mode, only the focused control is read, not adjacent controls. This is expected (and intended) behaviour. Only the text inside labels is exposed as the label of the control being labelled for accessibility purposes.

When a screen reader user is filling in a form using the tab key like this, they would expect to hit tab to move past the radio button to find further elements.
